src/rdkafka_cgrp.c

```diff
@@ -3285,6 +3291,16 @@ void rd_kafka_cgrp_handle_ConsumerGroupHeartbeat(rd_kafka_t *rk,
                     rkcg, "coordinator query");
         }
 
+        if (actions & RD_KAFKA_ERR_ACTION_SPECIAL &&
+            rkcg->rkcg_flags & RD_KAFKA_CGRP_F_SUBSCRIPTION &&
+            !(rkcg->rkcg_consumer_flags &
+              RD_KAFKA_CGRP_CONSUMER_F_ERROR_SENT)) {
+                rkcg->rkcg_consumer_flags |=
+                    RD_KAFKA_CGRP_CONSUMER_F_ERROR_SENT;
+                rd_kafka_consumer_err(
```

Comment:
Can we change this approach of sending error only once per subscription until the error is reset to continuously send the error for with some interval like we send in case of Disconnection.

- Its easy to miss a single log.
- User will never know if the problem is fixed or not until some message is received.